feedback.txt

```diff
@@ -1 +1 @@
-Give your first feedback.
+Thanks for this repo - devnischandra
```

Comment:
pls add your github username with the quote.

Reply:
sure..